src/index.html

```diff
@@ -12,6 +12,175 @@
   </head>
   <body>
     <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    autocomplete="off"
+    >
+
+      <fieldset class="form-set">
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input
+          type="name"
+          name="surname"
+          minlength="5"
+          maxlength="12"
+          required
+          tabindex="1"
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+          type="name"
+          name="name"
+          required
+          tabindex="2"
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?:
+          <input
+           type="number"
+           name="age"
+          min="1"
+          max="100"
+          value="12"
+          required
+          tabindex="3"
+          >
+        </label>
+
+        <label class="form-field">
+          Full data of birth:
+          <input
+          type="data"
+          name="birth"
+          placeholder="mm/dd/yyyy"
+          tabindex="4"
+          required
+          >
+        </label>
+
+        <label for="terms-and-conditions" class="form-field">
+          I accept the term of the agreement
+          <input
+           type="checkbox"
+           name="terms"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="form-set">
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          Email:
+          <input
+          type="email"
+          name="email"
+          placeholder="email@example.com"
+          tabindex="5"
+          required
+          >
+        </label>
+
+        <label class="form-field">Password:
+          <input
+          type="Password"
+          name="Password"
+          minlength="5"
+          maxlength="12"
+          tabindex="6"
+          required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-set">
+        <legend>An interesting fact about you?</legend>
+
+        <div class="form-field">
+          <label>Do you love cats?
+            <input
+            type="radio"
+            name="question"
+            >Yes
+            <input
+            type="radio"
+            name="question"
+            >No
+          </label>
+        </div>
```

Comment:
cover each input in a separate label to avoid this behavior

https://user-images.githubusercontent.com/70220667/234492004-d9a5ebe5-cdab-4590-9461-c0323d96e37e.mov